asyncscope.md

```diff
@@ -625,6 +630,115 @@ class Camera {
 }
 ```
 
+## Recursively spawning work until completion
+Below are three ways you could recursively spawn work on a scope using `let_with_async_scope` or `counting_scope`.
+
+### let_with_async_scope with spawn
+```cpp
+struct tree {
+  std::unique_ptr<tree> left;
+  std::unique_ptr<tree> right;
+  int data;
+};
+
+auto process(ex::scheduler auto sch, ex::counting_scope& scope, tree& t) noexcept {
+  return ex::schedule(sch) | then([sch, &]() {
+    if (t.left)
+      ex::spawn(scope, process(sch, scope, t.left.get()));
+    if (t.right)
+      ex::spawn(scope, process(sch, scope, t.right.get()));
+    do_stuff(t.data);
+  }), | ex::let_error([](auto& e) {
+    // log error
+    return just();
+  });
+}
+
+int main() {
+  ex::scheduler sch;
+  tree t = make_tree();
+  // let_with_async_scope will ensure all new work will be spawned on the
+  // scope and will not be joined until all work is finished.
+  // NOTE: Exceptions will not be surfaced to let_with_async_scope; exceptions
+  // will be handled by let_error instead.
+  this_thread::sync_wait(ex::let_with_async_scope([&, sch](auto scope) {
+	return process(sch, scope, t);
+  }));
+}
+```
+
+### let_with_async_scope with spawn_future
+```cpp
+struct tree {
+  std::unique_ptr<tree> left;
+  std::unique_ptr<tree> right;
+  int data;
+};
+
+auto process(ex::scheduler auto sch, ex::counting_scope& scope, tree& t) {
+    return ex::schedule(sch) | ex::let_value([sch, &]() {
+      ex::any_sender_of<> leftFut = ex::just();
+      ex::any_sender_of<> rightFut = ex::just();
```

Comment:
These should probably use the `unifex::` namespace since I don't think there's an `any_sender_of<>` in P2300.
```suggestion
      unifex::any_sender_of<> leftFut = ex::just();
      unifex::any_sender_of<> rightFut = ex::just();
```